src/test/java/plugins/FolderPluginTest.java

```diff
@@ -158,9 +158,11 @@ public void folderScopedCredentialsTest() {
         assertThat(console, containsRegexp(CRED_INJECTED_MESSAGE));
 
         job1.delete();
+        elasticSleep(1000);
         this.checkItemExists(job1, false);
```

Comment:
How about `waitFor(job1, Matcher.pageObjectDoesNotExist(), 1000)` and removing `checkItemExists`. The advantage, as recently pointed out in another PR, checking the URL presence any other way than using browser will break when used together with Jenkins authentication.